pkg/auth/signing/peers.go

```diff
@@ -39,6 +39,13 @@ type PublicKey struct {
 	Key  crypto.PublicKey
 }
 
+func SigneeFromFullIdentity(identity *identity.FullIdentity) Signee {
```

Comment:
You can use `SignerFromFullIdentity` since it implements both `Signee` and `Signer`.

Reply:
thanks, I missed that